### ISSUE TITLE
[Artistic drawing, minor issue] The NPC portraits seem to lack a bit colour hue/saturation I think

### ISSUE THREAD

**Issue by rubyFeedback** [Skitia/SkitiaNPCs#7]
One thing I noticed in regards to e. g. Emily (or whatever the name was of the archer,
my memory is awful so it may be other NPCs) is that the portrait is a bit "thin" on 
colours. I believe this is called saturation.

Now when I use gimp for image manipulation, increasing saturation does not always
lead to a better result. But I have had fairly ok-ish results for old fotos of reallife 
people, say, in the 1980s or early 1990s, where you could take a foto and the
image would appear on some small card as-is. These devices were pretty cool.

However had, I recently scanned old fotos from ~30 years ago, and the colours 
both have a red tinge, as well as lose colours over time, so I had to do a bit of
post-processing via gimp. For such fotos that post-processing worked fairly
well, including toning down red. Anyway - I don't know who was drawing these
portraits, but if there is ever a revisit of the portraits of the skitia NPCs, I
think some of them may benefit from a slightly "stronger" colour tone, e. g.
so that they don't look as if they lost colours. I noticed a similar with a
changed Aerie portrait of another mod (I think) where Aerie suddenly looked
as if the brightness was increased by +100%.

TL;DR: perhaps it can be considered to intensify the colours a little bit for the
skitia NPCs, if necessary.

**Comment by Skitia** [comment]
I can tweak it in photoshop or another program slightly for the brightness, but these were requested paintings from an artist I really cherish and I'm not too fond of making changes to their work. I'll probably just take this feedback towards the future if I ever request a new portrait to have some more options.
